alerts/alerts.libsonnet

```diff
@@ -0,0 +1,38 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gluster-utilization',
+        rules: [
+          {
+            alert: 'GlusterCpuPercentage',
+            expr: |||
+                  gluster_cpu_percentage > 0.8,
+            |||,
+            'for': '1m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Gluster - CPU Utilization more than 80%',
+            },
+          },
+          {
+            alert: 'GlusterVolumeUtilization',
+            expr: |||
+                  gluster_volume_utilization > 0.8,
+            |||,
+            'for': '1m',
```

Comment:
Is it really good to report within a minute if volume utilization crossed 80%. Can we think of using linear growth rate to decide on when to raise the alert?